pkg/appmanager/validateResources.go

```diff
@@ -155,6 +158,34 @@ func (appMgr *Manager) checkValidIngress(
 			return false, nil
 		}
 
+		// Validate url-rewrite annotations
+		if urlRewrite, ok := ing.ObjectMeta.Annotations[f5VsURLRewriteAnnotation]; ok {
+			if rsType == multiServiceIngressType {
+				urlRewriteMap := parseAppRootURLRewriteAnnotations(urlRewrite)
+				validateURLRewriteAnnotations(rsType, urlRewriteMap)
+			} else {
+				log.Warning("Single service ingress does not support url-rewrite annotation, not processing")
+			}
+		}
+
+		// Validate app-root annotations
+		if appRoot, ok := ing.ObjectMeta.Annotations[f5VsAppRootAnnotation]; ok {
+			appRootMap := parseAppRootURLRewriteAnnotations(appRoot)
+			if rsType == singleServiceIngressType {
+				if len(appRootMap) > 1 {
+					log.Warning("Single service ingress does not support multiple url-rewrite annotation values, not processing")
+				} else {
+					if _, ok := appRootMap["single"]; ok {
+						validateAppRootAnnotations(rsType, appRootMap)
+					} else {
+						log.Warningf("url-rewrite annotation: %s has targeted values which is not supported for single service ingress, not processing", appRoot)
```

Comment:
Wrong warning here, should be an app-root warning.

Reply:
Fixed.